ci/test/test_ci.py

```diff
@@ -39,5 +39,5 @@ async def wait_forever():
             log.info(f'returning {deploy_status} {failure_information}')
             return deploy_state, failure_information
 
-        deploy_state, failure_information = await asyncio.wait_for(wait_forever(), timeout=30 * 60)
+        deploy_state, failure_information = await wait_forever()
```

Comment:
the test job itself already has a timeout, so this is just a confusing redundancy